composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "booking-protect/insurance-hub-client",
+  "name": "urizev/booking-protect",
```

Comment:
could you change these back or exclude this commit from the pull?  I think you might have to revet and push or rebase and pick (up to you)